core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -138,7 +145,7 @@ public final boolean scheduleBuild(int quietPeriod, Cause c) {
         return Jenkins.getInstance().getQueue().schedule2(asJob(), quietPeriod, queueActions).getItem();
     }
 
-    private List<ParameterValue> getDefaultParametersValues() {
+    public List<ParameterValue> getDefaultParametersValues() {
```

Comment:
Yea, I should have made that public way back when I needed it, but I wasn't a core committer then and after I just forgot, and when I implemented the dockerhub trigger I remembered that I needed it, and forgot it again.

Basically the plugins wants to add their own parameters to the build, but not override the already defined parameters and add their default values to the build as well when triggering.